docs/DeveloperGuide.md

```diff
@@ -25,6 +40,159 @@
 |v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
 |v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
 
+### Main Menu
```

Comment:
Perhaps the capitalisation of the first letter of the headings' words can be standardised? I noticed that some headings only have the first word's letter capitalised (e.g. 'Design & implementation') while other headings capitalise every other word's first letter (e.g. 'Main Menu').